integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/cloud/v2/gcp/GcpCloudProvider.java

```diff
@@ -105,7 +113,7 @@ public StackAuthenticationEntity stackAuthentication(StackAuthenticationEntity s
     }
 
     @Override
-    public String getDefaultClusterDefinitionName() {
-        return GcpParameters.DEFAULT_CLUSTER_DEFINTION_NAME;
+    public String getClusterDefinitionName() {
+        return getTestParameter().getWithDefault(CommonCloudParameters.CLUSTER_DEFINITION_NAME, CommonCloudParameters.DEFAULT_CLUSTER_DEFINITION_NAME);
```

Comment:
Gcp has it's own 'DEFAULT_CLUSTER_DEFINTION_NAME', please use it. Applies to the other cloud providers too.